packages/core/src/components/Modal/footers/utils/getPropsForButton.ts

```diff
@@ -0,0 +1,14 @@
+import { ModalFooterActionProps } from "../ModalFooterBase/ModalFooterBase.types";
+import cx from "classnames";
+
+export function getPropsForButton(button?: ModalFooterActionProps, buttonClassName?: string) {
+  if (!button) return undefined;
+  const { tooltipProps, className, ...rest } = button;
+  return {
+    ...rest,
+    className: tooltipProps ? className : cx(className, buttonClassName),
+    tooltipProps: tooltipProps
+      ? { ...tooltipProps, referenceWrapperClassName: cx(tooltipProps.referenceWrapperClassName, buttonClassName) }
```

Comment:
Why do you need the referenceWrapperClassName? Does it add you another wrapping element?

Reply:
I need to add the button order class name to the first-wrapper, it can be either the button itself or the span of the Refable

Reply:
to allow the first child of the flex receive the "order" css attribute

Reply:
But shouldn't it be on the button element regardless?

Reply:
if the button's are wrapped with the span, then the span is the direct child of flex, so if the buttons themselves get the "order" css, it would have no effect. the order should be applied on the direct child of the flex container